src/styles/index.ts

```diff
@@ -5032,6 +5033,44 @@ const styles = (theme: ThemeColors) =>
             fontSize: variables.fontSizeNormal,
             fontWeight: FontUtils.fontWeight.bold,
         },
+
+        skeletonBackground: {
+            flex: 1,
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+        },
+
+        emptyStateForeground: (isSmallScreenWidth: boolean) => ({
+            justifyContent: 'center',
+            alignItems: 'center',
+            height: '100%',
+            padding: isSmallScreenWidth ? 24 : 0,
+        }),
+
+        emptyStateContent: (isSmallScreenWidth: boolean) => ({
+            width: isSmallScreenWidth ? '100%' : 400,
```

Comment:
Shouldn't `400` be a variable? Is it possible, that it will be used in different places?

Reply:
I don't think it will be reused somewhere